source/partial-deep.d.ts

```diff
@@ -1,5 +1,13 @@
 import type {BuiltIns} from './internal';
 
+/**
+The options for `PartialDeep`.
+@property recurseIntoArrays Whether to affect the individual elements of an array or tuple. (Default: true)
+*/
+interface PartialDeepOptions {
```

Comment:
You need to export this here and in the https://github.com/sindresorhus/type-fest/blob/main/index.d.ts

Reply:
Oh yeah, I don't know how I missed that. I'll add that later this week.